com.civicrm.hrjobroles/CRM/Hrjobroles/API/Query/ContactHrJobRolesSelect.php

```diff
@@ -129,6 +129,11 @@ private function addWhere(CRM_Utils_SQL_Select $customQuery) {
       $customQuery->where($conditions);
       unset($this->params['contact_id']);
     }
+
+    $dateRestriction = 'a.start_date <= "' . date('Y-m-d H:i:s') . '"';
+    $dateRestriction .= ' AND (a.end_date >= "'. date('Y-m-d H:i:s') . '" OR a.end_date IS NULL)';
+    $conditions[] = $dateRestriction;
+    $customQuery->where($conditions);
```

Comment:
There's another `$customQuery->where($conditions)` above, we can remove that since this will be called always.

Reply:
This is correct as this will process the `$conditions` together 👌 .